tools/apollo/create_or_update_organism.py

```diff
@@ -33,8 +45,19 @@
     if isinstance(org_cn, list):
         org_cn = org_cn[0]
 
-    # User must have an account
-    gx_user = AssertUser(wa.users.loadUsers(email=args.email))
+    # User must have an account, if not, create it
+    try:
+        gx_user = AssertUser(wa.users.loadUsers(email=args.email))
+    except Exception:
+        firstName = args.email
+        lastName = args.email
```

Comment:
doesn't apollo use something like `REMOTE_USER` as the name? I'm definitely forgetting since I haven't looked at it in some time.

Reply:
I think it does, though adding it in Galaxy would require removing the 'MAILDOMAIN'. Either we do it blindly (cut on the @), or we need to pass the conf value to the python script.
Either is fine, though I'm not sure if it is necessary (for consistency with existing account?)

Reply:
Yeah was just thinking for consistency. But not super necessary IMO. args.email is fine.